doc/api/async_hooks.md

```diff
@@ -0,0 +1,671 @@
+# Async Hooks
+
+
+> Stability: 2 - Stable
+
+
+The `async-hooks` module provides an API to register callbacks tracking the
+lifetime of asynchronous resources created inside a Node.js application.
+
+At its simplest form it can track metadata about each resource and log it when
+the program exits.
```

Comment:
This doesn't make much sense to me. The absolute simplest would be to log metadata about asynchronous resources when they occur.